integration/volume_create_integration_test.go

```diff
@@ -0,0 +1,216 @@
+
+	oriv1alpha1 "github.com/onmetal/cephlet/ori/volume/api/v1alpha1"
+	"github.com/onmetal/cephlet/pkg/api"
+	"github.com/onmetal/cephlet/pkg/omap"
+	metav1alpha1 "github.com/onmetal/onmetal-api/ori/apis/meta/v1alpha1"
+	onmetalv1alpha1 "github.com/onmetal/onmetal-api/ori/apis/volume/v1alpha1"
+)
+
+var _ = Describe("Create Volume", func() {
+	It("should create a volume", func(ctx SpecContext) {
+		createResp, err := volumeClient.CreateVolume(ctx, &onmetalv1alpha1.CreateVolumeRequest{
+			Volume: &onmetalv1alpha1.Volume{
+				Metadata: &metav1alpha1.ObjectMetadata{
+					Id: "foo",
+				},
+				Spec: &onmetalv1alpha1.VolumeSpec{
+					Class: "foo",
+					Resources: &onmetalv1alpha1.VolumeResources{
+						StorageBytes: 1024 * 1024 * 1024,
+					},
+				},
+			},
+		})
+		Expect(err).NotTo(HaveOccurred())
+		// Ensure the correct creation response
+		Expect(createResp).Should(SatisfyAll(
+			HaveField("Volume.Metadata.Id", Not(BeEmpty())),
+			HaveField("Volume.Spec.Image", Equal("")),
+			HaveField("Volume.Spec.Class", Equal("foo")),
+			HaveField("Volume.Spec.Resources.StorageBytes", Equal(uint64(1024*1024*1024))),
+			HaveField("Volume.Spec.Encryption", BeNil()),
+			HaveField("Volume.Status.State", Equal(onmetalv1alpha1.VolumeState_VOLUME_PENDING)),
+			HaveField("Volume.Status.Access", BeNil()),
+		))
+
+		// Ensure the correct image has been created inside the ceph cluster
+		image := &api.Image{}
+		Eventually(ctx, func() *api.Image {
+			oMap, err := ioctx.GetOmapValues(omap.OmapNameVolumes, "", createResp.Volume.Metadata.Id, 10)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(oMap).To(HaveKey(createResp.Volume.Metadata.Id))
+			Expect(json.Unmarshal(oMap[createResp.Volume.Metadata.Id], image)).NotTo(HaveOccurred())
+			return image
+		}).Should(SatisfyAll(
+			HaveField("Metadata.ID", Equal(createResp.Volume.Metadata.Id)),
+			HaveField("Metadata.Labels", HaveKeyWithValue(oriv1alpha1.ClassLabel, "foo")),
+			HaveField("Spec.Image", Equal("")),
+			HaveField("Spec.Size", Equal(uint64(1024*1024*1024))),
+			HaveField("Spec.Limits", SatisfyAll(
+				HaveKeyWithValue(api.IOPSBurstDurationLimit, int64(15)),
+				HaveKeyWithValue(api.WriteIOPSLimit, int64(100)),
+				HaveKeyWithValue(api.ReadBPSLimit, int64(100)),
+				HaveKeyWithValue(api.BPSLimit, int64(100)),
+				HaveKeyWithValue(api.ReadIOPSLimit, int64(100)),
+				HaveKeyWithValue(api.WriteBPSLimit, int64(100)),
+				HaveKeyWithValue(api.BPSBurstDurationLimit, int64(15)),
+				HaveKeyWithValue(api.IOPSlLimit, int64(100)),
+			)),
+			HaveField("Spec.SnapshotRef", BeNil()),
+			HaveField("Spec.Encryption.Type", api.EncryptionTypeUnencrypted),
+			HaveField("Spec.Encryption.EncryptedPassphrase", BeEmpty()),
+			HaveField("Status.State", Equal(api.ImageStateAvailable)),
+			HaveField("Status.Access", SatisfyAll(
+				HaveField("Monitors", os.Getenv("CEPH_MONITORS")),
+				HaveField("Handle", fmt.Sprintf("%s/%s", os.Getenv("CEPH_POOLNAME"), "img_"+createResp.Volume.Metadata.Id)),
```

Comment:
As described in the comment above: use the global variables here instead of reading the environment variables.